Source/Scene/Scene.js

```diff
@@ -1124,9 +1124,20 @@ define([
         if (command._dirty) {
             command._dirty = false;
             
+            var context = scene._context;
+            var derivedCommands = command.derivedCommands;
+
+            if (defined(scene.shadowMap)) {
+                derivedCommands.shadows = scene.shadowMap.createDerivedCommands(command, context, derivedCommands.shadows);
+            }
```

Comment:
I'm leaving this here for now. I'm changing it in a separate branch that is a merge between this one and `shadow-commands`.